src/fileio.jl

```diff
@@ -41,17 +67,18 @@ end
 function load(s::Stream{format"LAS"}; mmap=false)
     skiplasf(s)
     header = read(s, LasHeader)
+    n = header.extended_number_of_point_records > 0 ? Int(header.extended_number_of_point_records) : Int(header.records_count)
 
-    n = header.records_count
     pointtype = pointformat(header)
-
+    @info "Reading $(n) '$(pointtype)' points"
     if mmap
         pointsize = Int(header.data_record_length)
         pointbytes = Mmap.mmap(s.io, Vector{UInt8}, n*pointsize, position(s))
         pointdata = PointVector{pointtype}(pointbytes, pointsize)
```

Comment:
Doesn't the PointVector read all the data into an IOBuffer, defeating the purpose of memmapping?

Reply:
Isn't IO buffer a pointer to some place in memory. And we can used methods such as seek to extract data? I didn't touch any of the memory mapping code so will merge for now...